processing/src/main/java/io/druid/query/topn/TopNQueryQueryToolChest.java

```diff
@@ -291,6 +291,76 @@ public String apply(@Nullable AggregatorFactory input)
     return TYPE_REFERENCE;
   }
 
+  @VisibleForTesting
+  static Collection<PostAggregator> findPostAggregatorsForSort(
```

Comment:
This looks redundant to the already existing `private static List<PostAggregator> prunePostAggregators(TopNQuery query)` which calls `AggregatorUtil.pruneDependentPostAgg` (used for similar functionality in other parts of the topN system)